SpatialGDK/Documentation/content/get-started/gdk-and-starter-project.md

```diff
@@ -64,6 +64,11 @@ To run a cloud deployment, you need to prepare your server-worker and client-wor
 
 ### Congratulations!
 
-You've successfully set up and launched the Starter Project and the GDK! To start developing, we recommend following our Multiserver Shooter tutorial where you can try out the GDK’s development experience by adding a new feature to the Starter Project and test it across two servers.
+You've successfully set up and launched the Starter Project and the GDK! If you haven't already, we recommend following our Multiserver Shooter tutorial where you can try out the GDK’s development experience by adding a new feature to a project and test it across two servers.
```

Comment:
This reflects our new flow, Starter Project comes after tutorial

Reply:
Good catch!

Reply:
Changing to Testing

Reply:
Comma added